package.json

```diff
@@ -17,17 +17,17 @@
     "next": "^12.1.6",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "replicache": "^11.0.0",
-    "replicache-nextjs": "^0.4.0",
-    "replicache-react": "^2.9.0",
+    "replicache": "file:../../replicache-internal/replicache-12.0.0.tgz",
```

Comment:
Were you trying to test in preview? If so you'll have to add this file to the branch.

Reply:
No. I tested locally. Once pushed to npm I will update all these PRs